## 03 Aug 25 01:55 UTC

Success: false

### Versions

Terraform v1.12.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.2

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-smiling-chamois"
	            	[31mâ”‚[0m [0mService Name: "svsflkak-apim-service"
	            	[31mâ”‚[0m [0mApi: "ozsppnsl-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 27 Jul 25 01:41 UTC

Success: false

### Versions

Terraform v1.12.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.2

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-devoted-lab"
	            	[31mâ”‚[0m [0mService Name: "folqnmzs-apim-service"
	            	[31mâ”‚[0m [0mApi: "hbbctcfk-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 20 Jul 25 01:04 UTC

Success: false

### Versions

Terraform v1.12.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.2

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-keen-mullet"
	            	[31mâ”‚[0m [0mService Name: "gavsvgny-apim-service"
	            	[31mâ”‚[0m [0mApi: "ekczchiq-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 13 Jul 25 02:26 UTC

Success: false

### Versions

Terraform v1.12.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.2

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-intense-crane"
	            	[31mâ”‚[0m [0mService Name: "orapraij-apim-service"
	            	[31mâ”‚[0m [0mApi: "asmkzrqn-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 06 Jul 25 01:35 UTC

Success: false

### Versions

Terraform v1.12.1
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.2

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-informed-bulldog"
	            	[31mâ”‚[0m [0mService Name: "jhfgrxcv-apim-service"
	            	[31mâ”‚[0m [0mApi: "peyoxbcs-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 29 Jun 25 01:56 UTC

Success: false

### Versions

Terraform v1.12.1
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.2

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-close-hawk"
	            	[31mâ”‚[0m [0mService Name: "xxkmwhlf-apim-service"
	            	[31mâ”‚[0m [0mApi: "jkrhgbrm-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 15 Jun 25 02:24 UTC

Success: false

### Versions

Terraform v1.12.1
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.2

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-promoted-mosquito"
	            	[31mâ”‚[0m [0mService Name: "exelonsc-apim-service"
	            	[31mâ”‚[0m [0mApi: "atouuftv-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 08 Jun 25 01:45 UTC

Success: false

### Versions

Terraform v1.12.1
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.2

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-superb-mongoose"
	            	[31mâ”‚[0m [0mService Name: "xnurwhpe-apim-service"
	            	[31mâ”‚[0m [0mApi: "gfecdqpj-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 01 Jun 25 01:51 UTC

Success: false

### Versions

Terraform v1.12.1
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.2

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-improved-toucan"
	            	[31mâ”‚[0m [0mService Name: "zsronwjl-apim-service"
	            	[31mâ”‚[0m [0mApi: "rfqrkxac-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 25 May 25 01:43 UTC

Success: false

### Versions

Terraform v1.11.4
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.2

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-equal-ostrich"
	            	[31mâ”‚[0m [0mService Name: "ehvvdomt-apim-service"
	            	[31mâ”‚[0m [0mApi: "gklwbdzi-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 18 May 25 01:56 UTC

Success: false

### Versions

Terraform v1.11.4
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.2

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-fluent-kangaroo"
	            	[31mâ”‚[0m [0mService Name: "meovlpmk-apim-service"
	            	[31mâ”‚[0m [0mApi: "czrvagsv-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 11 May 25 01:52 UTC

Success: false

### Versions

Terraform v1.11.4
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.2

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-solid-stinkbug"
	            	[31mâ”‚[0m [0mService Name: "ohiphjho-apim-service"
	            	[31mâ”‚[0m [0mApi: "eobmswyx-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 04 May 25 02:31 UTC

Success: false

### Versions

Terraform v1.11.4
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.2

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-powerful-anchovy"
	            	[31mâ”‚[0m [0mService Name: "rmcmkwyg-apim-service"
	            	[31mâ”‚[0m [0mApi: "ptunghmn-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 27 Apr 25 01:22 UTC

Success: false

### Versions

Terraform v1.11.4
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.2

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-shining-marmot"
	            	[31mâ”‚[0m [0mService Name: "keejxkne-apim-service"
	            	[31mâ”‚[0m [0mApi: "mwukalnc-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 20 Apr 25 01:13 UTC

Success: false

### Versions

Terraform v1.11.4
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.1

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-actual-sunfish"
	            	[31mâ”‚[0m [0mService Name: "hdhvjlwr-apim-service"
	            	[31mâ”‚[0m [0mApi: "bdsfefus-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 13 Apr 25 02:23 UTC

Success: false

### Versions

Terraform v1.11.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.1

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-glowing-turtle"
	            	[31mâ”‚[0m [0mService Name: "haluqstd-apim-service"
	            	[31mâ”‚[0m [0mApi: "ptjckksq-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 06 Apr 25 01:46 UTC

Success: false

### Versions

Terraform v1.11.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.1

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-refined-anchovy"
	            	[31mâ”‚[0m [0mService Name: "etckqwjb-apim-service"
	            	[31mâ”‚[0m [0mApi: "pmdgumso-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 30 Mar 25 01:37 UTC

Success: false

### Versions

Terraform v1.11.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.1

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:52
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-deep-hedgehog"
	            	[31mâ”‚[0m [0mService Name: "erzawpgt-apim-service"
	            	[31mâ”‚[0m [0mApi: "oycyimfq-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 23 Mar 25 01:54 UTC

Success: false

### Versions

Terraform v1.11.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.1

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:51
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-equipped-sawfly"
	            	[31mâ”‚[0m [0mService Name: "gkwmgekz-apim-service"
	            	[31mâ”‚[0m [0mApi: "eodjmyvv-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 16 Mar 25 01:57 UTC

Success: false

### Versions

Terraform v1.11.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.1

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:51
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-causal-pony"
	            	[31mâ”‚[0m [0mService Name: "adhagtlv-apim-service"
	            	[31mâ”‚[0m [0mApi: "ghsghwoy-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 09 Mar 25 02:53 UTC

Success: false

### Versions

Terraform v1.10.5
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.1

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:50
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-enormous-moth"
	            	[31mâ”‚[0m [0mService Name: "pducpcbi-apim-service"
	            	[31mâ”‚[0m [0mApi: "ebvoecjw-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 02 Mar 25 04:58 UTC

Success: false

### Versions

Terraform v1.10.5
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.1
+ provider registry.terraform.io/hashicorp/random v3.7.1

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:50
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Service (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-skilled-antelope"
	            	[31mâ”‚[0m [0mService Name: "xsbwzgir-apim-service"): polling after CreateOrUpdate: polling failed: the Azure API returned the following error:
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0mStatus: "ActivationFailed"
	            	[31mâ”‚[0m [0mCode: ""
	            	[31mâ”‚[0m [0mMessage: "API Management service deployment into Virtual Network requires outbound connectivity to Azure Storage and Key Vault. Ensure outbound access to Microsoft.KeyVault and Microsoft.Storage service tags are allowed over port 443 in the NSG. Also ensure that Microsoft.KeyVault and Microsoft.Storage service endpoints are enabled on the subnet. For more details refer to https://aka.ms/apimvnet. VM has reported a failure when processing extension 'DSCConfiguration' (publisher 'Microsoft.Powershell' and type 'DSC'). Error message: 'DSC Configuration 'ApimDSC' completed with error(s). Following are the first few: Not enough memory resources are available to complete this operation.'. More information on troubleshooting is available at https://aka.ms/VMExtensionDSCWindowsTroubleshoot. "
	            	[31mâ”‚[0m [0mActivity Id: ""
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m---
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0mAPI Response:
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m----[start]----
	            	[31mâ”‚[0m [0m{"status":"Failed","error":{"code":"ActivationFailed","message":"API Management service deployment into Virtual Network requires outbound connectivity to Azure Storage and Key Vault. Ensure outbound access to Microsoft.KeyVault and Microsoft.Storage service tags are allowed over port 443 in the NSG. Also ensure that Microsoft.KeyVault and Microsoft.Storage service endpoints are enabled on the subnet. For more details refer to https://aka.ms/apimvnet. VM has reported a failure when processing extension 'DSCConfiguration' (publisher 'Microsoft.Powershell' and type 'DSC'). Error message: 'DSC Configuration 'ApimDSC' completed with error(s). Following are the first few: Not enough memory resources are available to complete this operation.'. More information on troubleshooting is available at https://aka.ms/VMExtensionDSCWindowsTroubleshoot. "}}
	            	[31mâ”‚[0m [0m-----[end]-----
	            	[31mâ”‚[0m [0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management.apim_service,
	            	[31mâ”‚[0m [0m  on main.tf line 18, in resource "azurerm_api_management" "apim_service":
	            	[31mâ”‚[0m [0m  18: resource "azurerm_api_management" "apim_service" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow
Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:135
	            				/usr/local/go/src/runtime/panic.go:629
	            				/usr/local/go/src/testing/testing.go:1006
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/test_runner.go:53
	            				/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:50
	Error:      	Received unexpected error:
	            	'terraform [destroy -auto-approve -input=false -lock=false]' unsuccessful after 5 retries
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 23 Feb 25 01:56 UTC

Success: false

### Versions

Terraform v1.10.5
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:50
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-hopeful-mutt"
	            	[31mâ”‚[0m [0mService Name: "bwqjyytr-apim-service"
	            	[31mâ”‚[0m [0mApi: "xnkvebqc-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 16 Feb 25 02:14 UTC

Success: false

### Versions

Terraform v1.10.5
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:50
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-related-dassie"
	            	[31mâ”‚[0m [0mService Name: "avaliadc-apim-service"
	            	[31mâ”‚[0m [0mApi: "ahesnsay-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 09 Feb 25 02:19 UTC

Success: false

### Versions

Terraform v1.10.5
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:50
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-famous-camel"
	            	[31mâ”‚[0m [0mService Name: "ksdefnyn-apim-service"
	            	[31mâ”‚[0m [0mApi: "usqzuhmg-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 02 Feb 25 02:16 UTC

Success: false

### Versions

Terraform v1.10.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:50
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-pet-chicken"
	            	[31mâ”‚[0m [0mService Name: "hypynste-apim-service"
	            	[31mâ”‚[0m [0mApi: "rdfscnzy-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 26 Jan 25 02:04 UTC

Success: false

### Versions

Terraform v1.10.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:50
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-enjoyed-walleye"
	            	[31mâ”‚[0m [0mService Name: "uhcxwkdi-apim-service"
	            	[31mâ”‚[0m [0mApi: "glkfdpmm-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 23 Jan 25 11:45 UTC

Success: false

### Versions

Terraform v1.10.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error

Error:
	Error Trace:	/home/runtimeuser/go/pkg/mod/github.com/gruntwork-io/terratest@v0.48.1/modules/terraform/apply.go:34
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:111
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:91
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:59
	            				/home/runtimeuser/go/pkg/mod/github.com/!azure/terraform-module-test-helper@v0.31.0/e2etest.go:55
	            				/src/test/e2e/quickstart_test.go:50
	Error:      	Received unexpected error:
	            	FatalError{Underlying: error while running command: exit status 1; [31mâ•·[0m[0m
	            	[31mâ”‚[0m [0m[1m[31mError: [0m[0m[1mcreating/updating Api (Subscription: "f7a632a5-49db-4c5e-9828-cd62cb753971"
	            	[31mâ”‚[0m [0mResource Group Name: "rg-just-barnacle"
	            	[31mâ”‚[0m [0mService Name: "haiseqgr-apim-service"
	            	[31mâ”‚[0m [0mApi: "ubpwibhz-api;rev=1"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: ValidationError: One or more fields contain incorrect values:[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ”‚[0m [0m[0m  with azurerm_api_management_api.api,
	            	[31mâ”‚[0m [0m  on main.tf line 56, in resource "azurerm_api_management_api" "api":
	            	[31mâ”‚[0m [0m  56: resource "azurerm_api_management_api" "api" [4m{[0m[0m
	            	[31mâ”‚[0m [0m
	            	[31mâ•µ[0m[0m}
	Test:       	Test_Quickstarts/quickstart/101-azure-api-management-create-with-api

FailNow

---

## 19 Jan 25 02:41 UTC

Success: false

### Versions

Terraform v1.10.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 12 Jan 25 03:01 UTC

Success: false

### Versions

Terraform v1.10.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 05 Jan 25 03:29 UTC

Success: false

### Versions

Terraform v1.10.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 29 Dec 24 02:43 UTC

Success: false

### Versions

Terraform v1.10.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 22 Dec 24 01:27 UTC

Success: false

### Versions

Terraform v1.10.2
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 15 Dec 24 03:01 UTC

Success: false

### Versions

Terraform v1.10.1
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 08 Dec 24 00:14 UTC

Success: false

### Versions

Terraform v1.9.6
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 01 Dec 24 00:16 UTC

Success: false

### Versions

Terraform v1.9.6
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 24 Nov 24 03:17 UTC

Success: false

### Versions

Terraform v1.9.6
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 17 Nov 24 02:10 UTC

Success: false

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 10 Nov 24 01:48 UTC

Success: false

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.117.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 03 Nov 24 02:51 UTC

Success: false

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.116.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 27 Oct 24 01:29 UTC

Success: false

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.116.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 20 Oct 24 01:38 UTC

Success: false

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.116.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 13 Oct 24 02:30 UTC

Success: false

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.116.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 06 Oct 24 01:36 UTC

Success: true

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.116.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 29 Sep 24 05:50 UTC

Success: true

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.116.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 29 Sep 24 02:50 UTC

Success: false

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.116.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 29 Sep 24 00:10 UTC

Success: false

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.116.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 27 Sep 24 08:02 UTC

Success: true

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.116.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 22 Sep 24 00:11 UTC

Success: false

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.116.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 18 Sep 24 03:29 UTC

Success: false

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.116.0
+ provider registry.terraform.io/hashicorp/random v3.6.3

### Error



---

## 08 Sep 24 03:04 UTC

Success: true

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.116.0
+ provider registry.terraform.io/hashicorp/random v3.6.2

### Error



---

## 01 Sep 24 02:34 UTC

Success: true

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.116.0
+ provider registry.terraform.io/hashicorp/random v3.6.2

### Error



---

## 25 Aug 24 02:51 UTC

Success: true

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.116.0
+ provider registry.terraform.io/hashicorp/random v3.6.2

### Error



---

## 18 Aug 24 03:12 UTC

Success: true

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.116.0
+ provider registry.terraform.io/hashicorp/random v3.6.2

### Error



---

## 11 Aug 24 02:45 UTC

Success: true

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.115.0
+ provider registry.terraform.io/hashicorp/random v3.6.2

### Error



---

## 04 Aug 24 02:51 UTC

Success: true

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.114.0
+ provider registry.terraform.io/hashicorp/random v3.6.2

### Error



---

## 01 Aug 24 03:44 UTC

Success: true

### Versions

Terraform v1.9.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.113.0
+ provider registry.terraform.io/hashicorp/random v3.6.2

### Error



---

